frontend/src/libs/components/header/header.tsx

```diff
@@ -2,15 +2,16 @@ import avatarImage from '~/assets/img/default-avatar-profile-icon.svg';
 import { Button, Link, Popover } from '~/libs/components/components.js';
 import { AppRoute } from '~/libs/enums/app-route.enum';
 import { useCallback } from '~/libs/hooks/hooks.js';
-import { type UserAuthResponse } from '~/packages/users/users.js';
+import { type ProjectUserEssence } from '~/packages/projects/projects.js';
 
 import styles from './styles.module.scss';
 
 type Properties = {
-  user: UserAuthResponse;
+  user: ProjectUserEssence | null;
```

Comment:
For what reason common header should know anything about any Project entity?
```suggestion
  user: UserAuthResponse;
```

Reply:
isn't resolved